src/templates/sitemap.ejs.xml

```diff
@@ -1,13 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
 <urlset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9 http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
 <% for (let url of urls ) { %>
     <url>
-        <loc>
-            https://almanac.httparchive.org/<%= url.url %>
-        </loc>
-        <% if (url.lastmod) { %>
-        <lastmod>
-            <%= url.lastmod %>
-        </lastmod>
+        <loc>https://almanac.httparchive.org/<%= url.url %></loc>
+        <% if (url.lastmod) { %><lastmod><%= url.lastmod %></lastmod>
         <% } %>
```

Comment:
Could put this on line 6 to compress the whitespace down further. Optional.

Reply:
??

Reply:
Ah got you. On it.